Dockerfile.image

```diff
@@ -45,6 +49,7 @@ RUN echo COMMIT=\"${IMAGE_COMMIT}\" >> /etc/os-release
 RUN echo IMAGE_REPO=\"${IMAGE_REPO}\" >> /etc/os-release
 RUN echo IMAGE_TAG=\"${IMAGE_TAG}\" >> /etc/os-release
 RUN echo IMAGE=\"${IMAGE_REPO}:${IMAGE_TAG}\" >> /etc/os-release
+RUN echo GRUB_ENTRY_NAME=\"Elemental ${IMAGE_TAG}\" >> /etc/os-release
```

Comment:
nit: I  know we used the have it alaways in this way, but I think having the version here is a bad idea. This is going to be used by active and passive grub options. After an update they are likely to be on a different version, however both will be named with the active version (the upgraded one), I find it pretty confusing.

Reply:
that is a good point, although take into consideration that this is only set here for github stuff. Our official releases are all done on OBS now so we probably wont have any releases here anymore as it doesn't make sense. 

I found that having the tag in there helps a lot when testing locally as you can see at a glance what was installed on booting, but I can easily drop, after all same info is available under os-release....maybe it would be even better to show that par under the motd...but that affects also downstream obs builds.

Or maybe I can just leave it out and have the Elemental in there, not really  care one way or the other

Reply:
just dropped, it. Makes no sense and its closer to OBS